bowtie/_cli.py

```diff
@@ -611,6 +611,14 @@ def summary(report: _report.Report, format: _F, show: str):
     """
     if show == "failures":
         results = report.worst_to_best()
+        exit_code = (
```

Comment:
This is a bit nonideal given it loops over the results multiple times (once here, once eventually when we're processing them). I haven't looked to see whether that's avoidable yet. Hopefully you did first?

Reply:
True. OK, let's keep it as is for now then, I want to replace lots of this presentation logic with something a bit more generic at some point. Thanks.